src/app/components/team/TeamBots.js

```diff
@@ -100,7 +100,7 @@ class TeamBots extends Component {
     const team = root.current_team;
 
     return (
-      <div className="team-bots">
```

Comment:
Why remove it? I find it useful to have these component-named classnames at the root element in a component when inspecting the DOM

Reply:
Well, for a couple reasons. 
1. it was not being used for testing or style changes, so it was kind of inert.
2. with the more simplified dom in this PR, when that wrapper div is present, it messes up the layout, since it creates a separation between some of the bot and the slack bot, resulting in no margin (see screenshot below). It could be added, but I think the added logic to create the margin isn't worth it, in my opinion

<img width="908" alt="Screenshot 2023-10-12 at 6 31 45 PM" src="https://github.com/meedan/check-web/assets/418001/c2b4b572-b6f2-45eb-9e29-c153c8f27d85">